examples/plot_DISCO_convolutions.py

```diff
@@ -0,0 +1,200 @@
+"""
+Training a SFNO on the spherical Shallow Water equations
+==========================================================
+
+In this example, we demonstrate how to use the small Spherical Shallow Water Equations example we ship with the package
+to train a Spherical Fourier-Neural Operator
+"""
+
+# %%
+# Preparation
+
+
+import os
+import torch
+import torch.nn as nn
+import numpy as np
+import math
+from functools import partial
+
+import imageio.v3 as iio
+
+from torch_harmonics.quadrature import legendre_gauss_weights, lobatto_weights, clenshaw_curtiss_weights
+
+import matplotlib.pyplot as plt
+
+cmap="inferno"
+device = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
+
+from neuralop.layers.discrete_continuous_convolution import DiscreteContinuousConv2d, DiscreteContinuousConvTranspose2d, EquidistantDiscreteContinuousConv2d, EquidistantDiscreteContinuousConvTranspose2d
+
+# %%
+# Let's start by loading an example image
+os.system("curl https://upload.wikimedia.org/wikipedia/commons/thumb/d/d3/Albert_Einstein_Head.jpg/360px-Albert_Einstein_Head.jpg -o ./einstein.jpg")
+
+nx = 90
+ny = 120
+
+img = iio.imread('./einstein.jpg')
```

Comment:
I don't think you need to add imageio to read in an image

Reply:
you are right - replaced it with matplotlib's image read function